Agent/Linux/ubuntu1804-README.md

```diff
@@ -0,0 +1,44 @@
+# Azure DevOps - Ubuntu 18.04 Agent
+
+The following sections provide a step-by-step guide to creating a Azure DevOps Agent Image using Packer, spin up an Agent using the Image and add it to the Azure DevOps Agent Pool.
+
+## Prepare your Azure DevOps Account
+
+1. Create an Azure DevOps Agent pool.
+
+2. Generate a Personal Access Token (PAT) for your Azure DevOps Organization. When generating the Personal Access Token (PAT), assign the following scopes:
+
+>a. Agent Pools - Read & Manage
+>b. Deployment Groups - Read & Manage
+
+3. Create a service principle and add it to your subscription as Owner. This is the subscription that will be used to host the image and Agent VM.
+
+4. Add the service principle to the subscription that will host the image and agent VMs with Owner role.
```

Comment:
principal, not principle, fix the typo please. Step number 4 is included in step number 3, maybe modify step 3 and include Create service principal only, step number 4 can be the RBAC assignment.

Reply:
Updated.